src/pfs/src/file.rs

```diff
@@ -167,6 +167,7 @@ impl<'a> Seek for File<'a> {
 }
 
 impl<'a> Read for File<'a> {
+    #[allow(clippy::uninit_vec)]
```

Comment:
Could you wrap the problematic lines inside `{}` then apply this to that block instead?